src/Maestro/Maestro.Web/Api/v2020_02_20/Controllers/PipelinesController.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Maestro.Data;
+using Microsoft.AspNetCore.ApiVersioning;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+
+namespace Maestro.Web.Api.v2020_02_20.Controllers
+{
+    /// <summary>
+    ///   We don't use Release Pipelines anymore.
+    /// </summary>
+    [Route("pipelines")]
+    [ApiVersion("2020-02-20")]
+    public class PipelinesController : v2018_07_16.Controllers.PipelinesController
```

Comment:
>v2018_07_16.Controllers.PipelinesController [](start = 39, length = 43)

You can just not inherit here and avoid having to write all the overrides.

Reply:
Absolutely. Thanks.